### PR TITLE
[4.1] adjustments in lang files

### LINKED ISSUE #2397: [4.1] adjustments in lang files
New language relevant PR in upstream repo: https://github.com/joomla/joomla-cms/pull/37535 Here are the upstream changes: 
<details>
<summary>Click to expand the diff!</summary>

```diff
diff --git a/administrator/language/en-GB/com_admin.ini b/administrator/language/en-GB/com_admin.ini
index 7e62e8b0cce3..189cbe9ef5b0 100644
--- a/administrator/language/en-GB/com_admin.ini
+++ b/administrator/language/en-GB/com_admin.ini
@@ -31,7 +31,7 @@ COM_ADMIN_GLOSSARY="Glossary"
 COM_ADMIN_HELP="Joomla! Help"
 COM_ADMIN_HELP_ARTICLES:_CATEGORIES="Articles: Categories"
 COM_ADMIN_HELP_ARTICLES:_EDIT="Articles: New/Edit"
-COM_ADMIN_HELP_ARTICLES:_FEATURED="Articles: Featured "
+COM_ADMIN_HELP_ARTICLES:_FEATURED="Articles: Featured"
 COM_ADMIN_HELP_ARTICLES:_NEW_OR_EDIT_CATEGORY="Articles: Categories - New/Edit"
 COM_ADMIN_HELP_ARTICLES="Articles"
 COM_ADMIN_HELP_BANNERS:_CATEGORIES="Banners: Categories"
diff --git a/administrator/language/en-GB/com_admin.sys.ini b/administrator/language/en-GB/com_admin.sys.ini
index 841707dfd94d..52fd51c94b29 100644
--- a/administrator/language/en-GB/com_admin.sys.ini
+++ b/administrator/language/en-GB/com_admin.sys.ini
@@ -3,7 +3,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-
 COM_ADMIN="System Information"
 COM_ADMIN_HELP_VIEW_DEFAULT_DESC="Get help on various pages in your Joomla administrator interface."
 COM_ADMIN_HELP_VIEW_DEFAULT_TITLE="Joomla! Help"
diff --git a/administrator/language/en-GB/com_contact.ini b/administrator/language/en-GB/com_contact.ini
index 9655ff66d714..5a5a2a1eaa44 100644
--- a/administrator/language/en-GB/com_contact.ini
+++ b/administrator/language/en-GB/com_contact.ini
@@ -27,7 +27,7 @@ COM_CONTACT_FIELD_CAPTCHA_LABEL="Allow Captcha on Contact"
 COM_CONTACT_FIELD_CATEGORY_LABEL="Select a Category"
 COM_CONTACT_FIELD_CONFIG_BANNED_EMAIL_DESC="Separate multiple email addresses with a semicolon."
 COM_CONTACT_FIELD_CONFIG_BANNED_EMAIL_LABEL="Banned Email Address"
-COM_CONTACT_FIELD_CONFIG_BANNED_SUBJECT_DESC=" Separate multiple subjects with a semicolon."
+COM_CONTACT_FIELD_CONFIG_BANNED_SUBJECT_DESC="Separate multiple subjects with a semicolon."
 COM_CONTACT_FIELD_CONFIG_BANNED_SUBJECT_LABEL="Banned Subject"
 COM_CONTACT_FIELD_CONFIG_BANNED_TEXT_DESC="Separate multiple words with a semicolon."
 COM_CONTACT_FIELD_CONFIG_BANNED_TEXT_LABEL="Banned Text"
diff --git a/administrator/language/en-GB/com_scheduler.ini b/administrator/language/en-GB/com_scheduler.ini
index 9d66db9b4876..4b4a87817972 100644
--- a/administrator/language/en-GB/com_scheduler.ini
+++ b/administrator/language/en-GB/com_scheduler.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 COM_SCHEDULER="Scheduled Tasks"
diff --git a/administrator/language/en-GB/com_scheduler.sys.ini b/administrator/language/en-GB/com_scheduler.sys.ini
index 90477b1ddde7..ff00c2fc960c 100644
--- a/administrator/language/en-GB/com_scheduler.sys.ini
+++ b/administrator/language/en-GB/com_scheduler.sys.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 COM_SCHEDULER="Scheduled Tasks"
@@ -11,6 +11,6 @@ COM_SCHEDULER_MANAGER_TASK_EDIT="Edit Task"
 COM_SCHEDULER_MANAGER_TASK_NEW="New Task"
 COM_SCHEDULER_MANAGER_TASKS="Tasks Manager"
 COM_SCHEDULER_MSG_MANAGE_NO_TASK_PLUGINS="There are no task types matching your query."
-COM_SCHEDULER_TASKS_VIEW_DEFAULT_DESC="Schedule and Manage Task Routines."
+COM_SCHEDULER_TASKS_VIEW_DEFAULT_DESC="Schedule and Manage Task Routines."
 COM_SCHEDULER_TASKS_VIEW_DEFAULT_TITLE="Scheduled Tasks"
 COM_SCHEDULER_XML_DESCRIPTION="Component for managing scheduled tasks and cronjobs (if supported by the server)."
diff --git a/administrator/language/en-GB/mod_custom.sys.ini b/administrator/language/en-GB/mod_custom.sys.ini
index 07a17df39ee2..551ba661f4ca 100644
--- a/administrator/language/en-GB/mod_custom.sys.ini
+++ b/administrator/language/en-GB/mod_custom.sys.ini
@@ -6,4 +6,3 @@
 MOD_CUSTOM="Custom"
 MOD_CUSTOM_LAYOUT_DEFAULT="Default"
 MOD_CUSTOM_XML_DESCRIPTION="This module allows you to create your own Module using a WYSIWYG editor."
-
diff --git a/administrator/language/en-GB/mod_latest.sys.ini b/administrator/language/en-GB/mod_latest.sys.ini
index 62ca0765affd..392ad5ad77e4 100644
--- a/administrator/language/en-GB/mod_latest.sys.ini
+++ b/administrator/language/en-GB/mod_latest.sys.ini
@@ -3,7 +3,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-
 MOD_LATEST="Articles - Latest"
 MOD_LATEST_LAYOUT_DEFAULT="Default"
 MOD_LATEST_XML_DESCRIPTION="This module shows a list of the most recently created Articles."
diff --git a/administrator/language/en-GB/mod_latestactions.sys.ini b/administrator/language/en-GB/mod_latestactions.sys.ini
index dd4a5de4624a..a2754e9250c7 100644
--- a/administrator/language/en-GB/mod_latestactions.sys.ini
+++ b/administrator/language/en-GB/mod_latestactions.sys.ini
@@ -3,7 +3,5 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-
 MOD_LATESTACTIONS="Action Logs - Latest"
 MOD_LATESTACTIONS_XML_DESCRIPTION="This module shows a list of the most recent actions."
-
diff --git a/administrator/language/en-GB/mod_logged.ini b/administrator/language/en-GB/mod_logged.ini
index 4e5fc50c76bc..0a94cf834342 100644
--- a/administrator/language/en-GB/mod_logged.ini
+++ b/administrator/language/en-GB/mod_logged.ini
@@ -3,7 +3,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-
 MOD_LOGGED="Logged-in Users"
 MOD_LOGGED_FIELD_COUNT_LABEL="Users to Display"
 MOD_LOGGED_LOGOUT="Logout"
diff --git a/administrator/language/en-GB/mod_logged.sys.ini b/administrator/language/en-GB/mod_logged.sys.ini
index 84e33bd2131f..1f5af276312f 100644
--- a/administrator/language/en-GB/mod_logged.sys.ini
+++ b/administrator/language/en-GB/mod_logged.sys.ini
@@ -3,7 +3,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-
 MOD_LOGGED="Logged-in Users"
 MOD_LOGGED_LAYOUT_DEFAULT="Default"
 MOD_LOGGED_XML_DESCRIPTION="This module shows a list of the Logged-in Users."
diff --git a/administrator/language/en-GB/plg_system_httpheaders.ini b/administrator/language/en-GB/plg_system_httpheaders.ini
index ee90100271ba..5ef05e0310cc 100644
--- a/administrator/language/en-GB/plg_system_httpheaders.ini
+++ b/administrator/language/en-GB/plg_system_httpheaders.ini
@@ -42,4 +42,3 @@ PLG_SYSTEM_HTTPHEADERS_POSTINSTALL_INTRODUCTION_TITLE="HTTP Security Headers"
 PLG_SYSTEM_HTTPHEADERS_REFERRERPOLICY="<a href='https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy' target='_blank' rel='noopener noreferrer'>Referrer-Policy</a>" ; Do not translate
 PLG_SYSTEM_HTTPHEADERS_XFRAMEOPTIONS="<a href='https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options' target='_blank' rel='noopener noreferrer'>X-Frame-Options</a>" ; Do not translate
 PLG_SYSTEM_HTTPHEADERS_XML_DESCRIPTION="This Plugin helps you to set the HTTP Security Headers" ; Please do not translate 'HTTP Security Headers'
-
diff --git a/administrator/language/en-GB/plg_task_checkfiles.ini b/administrator/language/en-GB/plg_task_checkfiles.ini
index a3fa03405da9..8436f26e9364 100644
--- a/administrator/language/en-GB/plg_task_checkfiles.ini
+++ b/administrator/language/en-GB/plg_task_checkfiles.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_CHECK_FILES="Task - Check Files"
diff --git a/administrator/language/en-GB/plg_task_checkfiles.sys.ini b/administrator/language/en-GB/plg_task_checkfiles.sys.ini
index 11efd630767f..1de2b3dd0127 100644
--- a/administrator/language/en-GB/plg_task_checkfiles.sys.ini
+++ b/administrator/language/en-GB/plg_task_checkfiles.sys.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_CHECK_FILES="Task - Check Files"
diff --git a/administrator/language/en-GB/plg_task_demotasks.ini b/administrator/language/en-GB/plg_task_demotasks.ini
index d7c6c8459020..b2d276c78a97 100644
--- a/administrator/language/en-GB/plg_task_demotasks.ini
+++ b/administrator/language/en-GB/plg_task_demotasks.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_DEMO_TASKS="Task - Demo Tasks!"
diff --git a/administrator/language/en-GB/plg_task_demotasks.sys.ini b/administrator/language/en-GB/plg_task_demotasks.sys.ini
index eae925bdd128..6e691072a04b 100644
--- a/administrator/language/en-GB/plg_task_demotasks.sys.ini
+++ b/administrator/language/en-GB/plg_task_demotasks.sys.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_DEMO_TASKS="Task - Demo Tasks!"
diff --git a/administrator/language/en-GB/plg_task_requests.ini b/administrator/language/en-GB/plg_task_requests.ini
index 293644e86a90..8b47db87f554 100644
--- a/administrator/language/en-GB/plg_task_requests.ini
+++ b/administrator/language/en-GB/plg_task_requests.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_REQUESTS="Task - Requests"
diff --git a/administrator/language/en-GB/plg_task_requests.sys.ini b/administrator/language/en-GB/plg_task_requests.sys.ini
index 148a036d3167..40dd4aed4036 100644
--- a/administrator/language/en-GB/plg_task_requests.sys.ini
+++ b/administrator/language/en-GB/plg_task_requests.sys.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_REQUESTS="Task - Requests"
diff --git a/administrator/language/en-GB/plg_task_sitestatus.ini b/administrator/language/en-GB/plg_task_sitestatus.ini
index 4d57ca42c11e..d71c94e9c741 100644
--- a/administrator/language/en-GB/plg_task_sitestatus.ini
+++ b/administrator/language/en-GB/plg_task_sitestatus.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_SITE_STATUS="Task - Site Status"
diff --git a/administrator/language/en-GB/plg_task_sitestatus.sys.ini b/administrator/language/en-GB/plg_task_sitestatus.sys.ini
index eb0735664baa..ffbf2169f7d0 100644
--- a/administrator/language/en-GB/plg_task_sitestatus.sys.ini
+++ b/administrator/language/en-GB/plg_task_sitestatus.sys.ini
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
-; GNU General Public License version 2 or later; see LICENSE.txt
+; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_TASK_SITE_STATUS="Task - Site Status"
diff --git a/language/en-GB/com_ajax.ini b/language/en-GB/com_ajax.ini
index 887b10f2c4f0..aea050bbe167 100644
--- a/language/en-GB/com_ajax.ini
+++ b/language/en-GB/com_ajax.ini
@@ -3,7 +3,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-
 COM_AJAX="Ajax Interface"
 COM_AJAX_FILE_NOT_EXISTS="The file at %s does not exist."
 COM_AJAX_METHOD_NOT_EXISTS="Method %s does not exist."
diff --git a/language/en-GB/joomla.ini b/language/en-GB/joomla.ini
index 11359813312a..f0a546619e06 100644
--- a/language/en-GB/joomla.ini
+++ b/language/en-GB/joomla.ini
@@ -11,8 +11,8 @@ JERROR_PARSING_LANGUAGE_FILE="&#160;: error(s) in line(s) %s"
 
 ERROR="Error"
 INFO="Info"
-NOTICE="Notice"
 MESSAGE="Message"
+NOTICE="Notice"
 WARNING="Warning"
 
 J1="1"
diff --git a/language/en-GB/lib_joomla.sys.ini b/language/en-GB/lib_joomla.sys.ini
index 2989f8e00b55..413633dbe986 100644
--- a/language/en-GB/lib_joomla.sys.ini
+++ b/language/en-GB/lib_joomla.sys.ini
@@ -5,4 +5,3 @@
 
 LIB_JOOMLA="Joomla! Platform"
 LIB_JOOMLA_XML_DESCRIPTION="The Joomla! Platform is the Core of the Joomla! Content Management System."
-
diff --git a/language/en-GB/lib_simplepie.sys.ini b/language/en-GB/lib_simplepie.sys.ini
index eee9c05c4d68..a4ebb029c69e 100644
--- a/language/en-GB/lib_simplepie.sys.ini
+++ b/language/en-GB/lib_simplepie.sys.ini
@@ -4,4 +4,3 @@
 ; Note : All ini files need to be saved as UTF-8
 
 LIB_SIMPLEPIE_XML_DESCRIPTION="PHP based RSS and Atom Feed Framework."
-
diff --git a/language/en-GB/mod_articles_category.sys.ini b/language/en-GB/mod_articles_category.sys.ini
index d237edf0e8bc..dbeffc1a2ae8 100644
--- a/language/en-GB/mod_articles_category.sys.ini
+++ b/language/en-GB/mod_articles_category.sys.ini
@@ -6,4 +6,3 @@
 MOD_ARTICLES_CATEGORY="Articles - Category"
 MOD_ARTICLES_CATEGORY_LAYOUT_DEFAULT="Default"
 MOD_ARTICLES_CATEGORY_XML_DESCRIPTION="This module displays a list of articles from one or more categories."
-
diff --git a/language/en-GB/mod_articles_latest.sys.ini b/language/en-GB/mod_articles_latest.sys.ini
index ffccc2980a44..29499eaacd38 100644
--- a/language/en-GB/mod_articles_latest.sys.ini
+++ b/language/en-GB/mod_articles_latest.sys.ini
@@ -6,4 +6,3 @@
 MOD_ARTICLES_LATEST="Articles - Latest"
 MOD_ARTICLES_LATEST_LAYOUT_DEFAULT="Default"
 MOD_LATEST_NEWS_XML_DESCRIPTION="This module shows a list of the most recently published and current Articles."
-
diff --git a/language/en-GB/mod_articles_news.sys.ini b/language/en-GB/mod_articles_news.sys.ini
index 71c6ae15d9a3..d618f6d886ba 100644
--- a/language/en-GB/mod_articles_news.sys.ini
+++ b/language/en-GB/mod_articles_news.sys.ini
@@ -6,4 +6,3 @@
 MOD_ARTICLES_NEWS="Articles - Newsflash"
 MOD_ARTICLES_NEWS_LAYOUT_DEFAULT="Default"
 MOD_ARTICLES_NEWS_XML_DESCRIPTION="The Newsflash Module will display a fixed number of articles from a specific category."
-
diff --git a/language/en-GB/mod_articles_popular.sys.ini b/language/en-GB/mod_articles_popular.sys.ini
index a0830faf6447..d57521510e7a 100644
--- a/language/en-GB/mod_articles_popular.sys.ini
+++ b/language/en-GB/mod_articles_popular.sys.ini
@@ -6,4 +6,3 @@
 MOD_ARTICLES_POPULAR="Articles - Most Read"
 MOD_ARTICLES_POPULAR_LAYOUT_DEFAULT="Default"
 MOD_POPULAR_XML_DESCRIPTION="This module shows a list of the published Articles which have the highest number of page views."
-
diff --git a/language/en-GB/mod_banners.sys.ini b/language/en-GB/mod_banners.sys.ini
index 0e53f9b7cd52..05095b299cd8 100644
--- a/language/en-GB/mod_banners.sys.ini
+++ b/language/en-GB/mod_banners.sys.ini
@@ -6,4 +6,3 @@
 MOD_BANNERS="Banners"
 MOD_BANNERS_LAYOUT_DEFAULT="Default"
 MOD_BANNERS_XML_DESCRIPTION="The Banner Module displays the active Banners from the Component."
-
diff --git a/language/en-GB/mod_breadcrumbs.sys.ini b/language/en-GB/mod_breadcrumbs.sys.ini
index 133c043afb41..71486b50b418 100644
--- a/language/en-GB/mod_breadcrumbs.sys.ini
+++ b/language/en-GB/mod_breadcrumbs.sys.ini
@@ -6,4 +6,3 @@
 MOD_BREADCRUMBS="Breadcrumbs"
 MOD_BREADCRUMBS_LAYOUT_DEFAULT="Default"
 MOD_BREADCRUMBS_XML_DESCRIPTION="This module displays the Breadcrumbs."
-
diff --git a/language/en-GB/mod_custom.sys.ini b/language/en-GB/mod_custom.sys.ini
index 07a17df39ee2..551ba661f4ca 100644
--- a/language/en-GB/mod_custom.sys.ini
+++ b/language/en-GB/mod_custom.sys.ini
@@ -6,4 +6,3 @@
 MOD_CUSTOM="Custom"
 MOD_CUSTOM_LAYOUT_DEFAULT="Default"
 MOD_CUSTOM_XML_DESCRIPTION="This module allows you to create your own Module using a WYSIWYG editor."
-
diff --git a/language/en-GB/mod_feed.sys.ini b/language/en-GB/mod_feed.sys.ini
index 85e1c07b704c..db0caffe7fce 100644
--- a/language/en-GB/mod_feed.sys.ini
+++ b/language/en-GB/mod_feed.sys.ini
@@ -6,4 +6,3 @@
 MOD_FEED="Feed Display"
 MOD_FEED_LAYOUT_DEFAULT="Default"
 MOD_FEED_XML_DESCRIPTION="This module allows the displaying of a syndicated feed."
-
diff --git a/language/en-GB/mod_footer.sys.ini b/language/en-GB/mod_footer.sys.ini
index 9294b58e5b37..a8c0f1c44e38 100644
--- a/language/en-GB/mod_footer.sys.ini
+++ b/language/en-GB/mod_footer.sys.ini
@@ -6,4 +6,3 @@
 MOD_FOOTER="Footer"
 MOD_FOOTER_LAYOUT_DEFAULT="Default"
 MOD_FOOTER_XML_DESCRIPTION="This module shows the Joomla! copyright information."
-
diff --git a/language/en-GB/mod_login.sys.ini b/language/en-GB/mod_login.sys.ini
index 8122d48642c9..61d68e506fda 100644
--- a/language/en-GB/mod_login.sys.ini
+++ b/language/en-GB/mod_login.sys.ini
@@ -6,4 +6,3 @@
 MOD_LOGIN="Login"
 MOD_LOGIN_LAYOUT_DEFAULT="Default"
 MOD_LOGIN_XML_DESCRIPTION="This module displays a username and password login form. It also displays a link to retrieve a forgotten password. If user registration is enabled (in Users &rarr; Manage &rarr; Options), another link will be shown to enable self-registration for users."
-
diff --git a/language/en-GB/mod_menu.sys.ini b/language/en-GB/mod_menu.sys.ini
index 07085d26a319..a80b981d3bb2 100644
--- a/language/en-GB/mod_menu.sys.ini
+++ b/language/en-GB/mod_menu.sys.ini
@@ -7,4 +7,3 @@ MOD_MENU="Menu"
 MOD_MENU_LAYOUT_COLLAPSE-DEFAULT="Collapsible Default Menu"
 MOD_MENU_LAYOUT_DEFAULT="Default"
 MOD_MENU_XML_DESCRIPTION="This module displays a menu on the Frontend."
-
diff --git a/language/en-GB/mod_random_image.sys.ini b/language/en-GB/mod_random_image.sys.ini
index 6438115eebb6..9c03ae47f269 100644
--- a/language/en-GB/mod_random_image.sys.ini
+++ b/language/en-GB/mod_random_image.sys.ini
@@ -6,4 +6,3 @@
 MOD_RANDOM_IMAGE="Random Image"
 MOD_RANDOM_IMAGE_LAYOUT_DEFAULT="Default"
 MOD_RANDOM_IMAGE_XML_DESCRIPTION="This module displays a random image from your chosen folder."
-
diff --git a/language/en-GB/mod_stats.sys.ini b/language/en-GB/mod_stats.sys.ini
index 9a38cd9aca14..fb88027d3ac5 100644
--- a/language/en-GB/mod_stats.sys.ini
+++ b/language/en-GB/mod_stats.sys.ini
@@ -6,4 +6,3 @@
 MOD_STATS="Statistics"
 MOD_STATS_LAYOUT_DEFAULT="Default"
 MOD_STATS_XML_DESCRIPTION="The Statistics Module shows information about your server installation together with statistics on the website users and the number of Articles in your database."
-
diff --git a/language/en-GB/mod_syndicate.sys.ini b/language/en-GB/mod_syndicate.sys.ini
index 3b62eed7284a..359a7d9189bc 100644
--- a/language/en-GB/mod_syndicate.sys.ini
+++ b/language/en-GB/mod_syndicate.sys.ini
@@ -6,4 +6,3 @@
 MOD_SYNDICATE="Syndication Feeds"
 MOD_SYNDICATE_LAYOUT_DEFAULT="Default"
 MOD_SYNDICATE_XML_DESCRIPTION="Smart Syndication Module that creates a Syndicated Feed for the page where the Module is displayed."
-
diff --git a/language/en-GB/mod_tags_similar.sys.ini b/language/en-GB/mod_tags_similar.sys.ini
index d948e650d80a..51c0e35211bb 100644
--- a/language/en-GB/mod_tags_similar.sys.ini
+++ b/language/en-GB/mod_tags_similar.sys.ini
@@ -6,4 +6,3 @@
 MOD_TAGS_SIMILAR="Tags - Similar"
 MOD_TAGS_SIMILAR_LAYOUT_DEFAULT="Default"
 MOD_TAGS_SIMILAR_XML_DESCRIPTION="The Similar Tags Module displays links to other items with similar tags. The closeness of the match can be specified."
-
diff --git a/language/en-GB/mod_users_latest.sys.ini b/language/en-GB/mod_users_latest.sys.ini
index 51e3f7c8b3ba..714dcb14d977 100644
--- a/language/en-GB/mod_users_latest.sys.ini
+++ b/language/en-GB/mod_users_latest.sys.ini
@@ -6,4 +6,3 @@
 MOD_USERS_LATEST="Latest Users"
 MOD_USERS_LATEST_LAYOUT_DEFAULT="Default"
 MOD_USERS_LATEST_XML_DESCRIPTION="This module displays the latest registered users."
-
diff --git a/language/en-GB/mod_whosonline.sys.ini b/language/en-GB/mod_whosonline.sys.ini
index c134d26a973f..ed8f0fe5e61a 100644
--- a/language/en-GB/mod_whosonline.sys.ini
+++ b/language/en-GB/mod_whosonline.sys.ini
@@ -6,4 +6,3 @@
 MOD_WHOSONLINE="Who's Online"
 MOD_WHOSONLINE_LAYOUT_DEFAULT="Default"
 MOD_WHOSONLINE_XML_DESCRIPTION="The Who's Online Module displays the number of Anonymous Users (Guests) and Registered Users (users logged-in) that are currently accessing the website."
-

```
</details>

### PR DESCRIPTION
**Pull request by heelc29**
Pull Request für Issue #2397 .

### Zusammenfassung der Änderungen

- remove empty lines
- remove unnecessary spaces

### Wo wird der Sprachstring angezeigt / Wie kann getestet werden

code review

### CODE REVIEW

File: administrator/language/de-DE/com_joomlaupdate.sys.ini [hunk 1]
```diff
@@ -9,4 +9,3 @@ COM_JOOMLAUPDATE_DEFAULT_VIEW_DEFAULT_DESC="Nach Joomla-Updates suchen und die W
 COM_JOOMLAUPDATE_DEFAULT_VIEW_DEFAULT_TITLE="Joomla-Updates anzeigen"
 COM_JOOMLAUPDATE_JOOMLAUPDATE_VIEW_DISPLAY_BADGE="Badge anzeigen"
 COM_JOOMLAUPDATE_XML_DESCRIPTION="Ein Ein-Klick-Update von Joomla! zur neusten Version."
```
Comment (tecpromotion) [inline administrator/language/de-DE/com_joomlaupdate.sys.ini]:
```suggestion
COM_JOOMLAUPDATE_XML_DESCRIPTION="Ein Ein-Klick-Update von Joomla! zur neusten Version."

```

File: administrator/language/de-DE/mod_privacy_status.ini [hunk 2]
```diff
@@ -8,4 +8,3 @@ MOD_PRIVACY_STATUS="Datenschutzstatus"
 MOD_PRIVACY_STATUS_CHECK_DATABASE_CONNECTION_ENCRYPTION_DISABLED="Die Verschlüsselung der Datenbankverbindung ist nicht aktiviert."
 MOD_PRIVACY_STATUS_CHECK_DATABASE_CONNECTION_ENCRYPTION_ENABLED="Die Verschlüsselung der Datenbankverbindung ist aktiviert.<br><small>Alle Datenbankverbindungen werden mit „%s“ verschlüsselt</small>."
 MOD_PRIVACY_STATUS_XML_DESCRIPTION="Dieses Modul für den Datenschutzstatus zeigt Informationen über den Datenschutzstatus Ihrer Website an."
```
Comment (tecpromotion) [inline administrator/language/de-DE/mod_privacy_status.ini]:
```suggestion
MOD_PRIVACY_STATUS_XML_DESCRIPTION="Dieses Modul für den Datenschutzstatus zeigt Informationen über den Datenschutzstatus Ihrer Website an."

```

File: language/de-DE/mod_articles_archive.ini [hunk 3]
```diff
@@ -8,4 +8,3 @@ MOD_ARTICLES_ARCHIVE="Beiträge - Archiv"
 MOD_ARTICLES_ARCHIVE_DATE="%1$s, %2$s"
 MOD_ARTICLES_ARCHIVE_FIELD_COUNT_LABEL="# Monate"
 MOD_ARTICLES_ARCHIVE_XML_DESCRIPTION="Dieses Modul zeigt eine Liste der Kalendermonate, die archivierte Beiträge enthalten. Nach dem Archivieren von Beiträgen wird diese Liste automatisch generiert."
```
Comment (tecpromotion) [inline language/de-DE/mod_articles_archive.ini]:
```suggestion
MOD_ARTICLES_ARCHIVE_XML_DESCRIPTION="Dieses Modul zeigt eine Liste der Kalendermonate, die archivierte Beiträge enthalten. Nach dem Archivieren von Beiträgen wird diese Liste automatisch generiert."

```

File: language/de-DE/mod_custom.ini [hunk 4]
```diff
@@ -9,4 +9,3 @@ MOD_CUSTOM_FIELD_BACKGROUNDIMAGE_LABEL="Hintergrundbild auswählen"
 MOD_CUSTOM_FIELD_PREPARE_CONTENT_DESC="Bereitet die Modulinhalte mit den aktivierten Inhaltsplugins auf (optional). Plugins funktionieren somit nicht nur in Beiträgen, sondern auch in diesem Modul."
 MOD_CUSTOM_FIELD_PREPARE_CONTENT_LABEL="Inhalte vorbereiten"
 MOD_CUSTOM_XML_DESCRIPTION="Dieses Modul ermöglicht es ein eigenes Modul mithilfe eines WYSIWYG-Editors zu erstellen."
```
Comment (tecpromotion) [inline language/de-DE/mod_custom.ini]:
```suggestion
MOD_CUSTOM_XML_DESCRIPTION="Dieses Modul ermöglicht es ein eigenes Modul mithilfe eines WYSIWYG-Editors zu erstellen."

```

File: language/de-DE/lib_phpass.sys.ini [hunk 5]
```diff
@@ -6,4 +6,3 @@
 
 LIB_PHPASS="PHPass"
 LIB_PHPASS_XML_DESCRIPTION="PHPass ist ein portables Passwort-Hashing-Framework für den Einsatz in PHP-Anwendungen. Die bevorzugte (sicherste) von phpass unterstützte Hashing-Methode ist die OpenBSD-style bcrypt (in PHP als CRYPT_BLOWFISH bekannt), mit einem Fallback auf BSDI-style extended DES-basierte Hashes (in PHP als CRYPT_EXT_DES bekannt), und einem letztlichen Fallback auf eine MD5-basierte variable Iterationszählung des Passwort-Hashing-Verfahrens, die in phpass selbst implementiert ist."
```
Comment (tecpromotion) [inline language/de-DE/lib_phpass.sys.ini]:
```suggestion
LIB_PHPASS_XML_DESCRIPTION="PHPass ist ein portables Passwort-Hashing-Framework für den Einsatz in PHP-Anwendungen. Die bevorzugte (sicherste) von phpass unterstützte Hashing-Methode ist die OpenBSD-style bcrypt (in PHP als CRYPT_BLOWFISH bekannt), mit einem Fallback auf BSDI-style extended DES-basierte Hashes (in PHP als CRYPT_EXT_DES bekannt), und einem letztlichen Fallback auf eine MD5-basierte variable Iterationszählung des Passwort-Hashing-Verfahrens, die in phpass selbst implementiert ist."

```

**Review by tecpromotion — APPROVED** [review]

**Comment by tecpromotion** [comment]
@heelc29 please always end files with a blank line.
Thanks!

**Comment by heelc29** [comment]
> @heelc29 please always end files with a blank line. Thanks!

@tecpromotion

There are **two** blank linkes
image

In the other files are only **one** blank line
image

**Comment by tecpromotion** [comment]
fixed with #2399 
sorry

**Pull request merged**